src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -200,8 +200,10 @@ def __init__(self, events_path: str) -> None:
                     begin_msg = msg  # store first one
                 end_msg = msg  # store last one
 
+        # Ensure that trace data includes one at least. If there is no message in trace data, assertion failed.
```

Comment:
Sorry for my mistake. This cause the error on flake8. 
```suggestion
        # Ensure that trace data includes one at least.
        # If there is no message in trace data, assertion failed.
```